sdk/keyvault/azure-security-keyvault-keys/test/samples/sample1-hello-world/sample1_hello_world.cpp

```diff
@@ -30,13 +30,14 @@ using namespace Azure::Security::KeyVault::Keys;
 
 int main()
 {
-  auto tenantId = std::getenv("AZURE_TENANT_ID");
-  auto clientId = std::getenv("AZURE_CLIENT_ID");
-  auto clientSecret = std::getenv("AZURE_CLIENT_SECRET");
+  auto const tenantId = std::string(std::getenv("AZURE_TENANT_ID"));
```

Comment:
In samples at least, and in future in all code, let's use the `const X` instead of `X const`.
In this PR, I've seen instances of `auto const` and `uint8_t const` in samples, I recommend to update them.

Reply:
really?   why?   :(   

I would expect any user to know that it's the same.